Author: Pino Toscano <pino@debian.org>
Description: Disable fetchertest
 It seems it actually runs the fetchers, hitting the network.
Last-Update: 2021-08-21
Forwarded: no

--- a/src/tests/CMakeLists.txt
+++ b/src/tests/CMakeLists.txt
@@ -459,6 +459,8 @@ if(${KF5KIO_VERSION} VERSION_GREATER "5.
   target_link_libraries(fetcherstest KF5::KIOGui)
 endif(${KF5KIO_VERSION} VERSION_GREATER "5.18.0")
 
+IF(BUILD_FETCHER_TESTS)
+
 # the fetch library really depends on the GUI static library
 # but rather than linking all of that in, selectively choose the
 # specific classes used
@@ -478,6 +480,8 @@ ecm_add_test(fetchertest.cpp
     LINK_LIBRARIES fetch translatorstest ${TELLICO_TEST_LIBS} KF5::XmlGui KF5::SonnetCore KF5::SonnetUi
 )
 
+ENDIF(BUILD_FETCHER_TESTS)
+
 # the PDF importer uses CrossRefFetcher, so include the test near the fetchers section
 ecm_add_test(pdftest.cpp
              ../translators/pdfimporter.cpp
